requirements/requirements.in

```diff
@@ -25,6 +25,7 @@ django-split-settings
 django-taggit
 djangorestframework==3.13.1
 djangorestframework-yaml
+filelock
```

Comment:
You've got to do the pip-compile dance.  https://github.com/ansible/awx/blob/devel/requirements/README.md#upgrading-or-adding-select-libraries

Reply:
ugg... done